package.json

```diff
@@ -1,12 +1,12 @@
 {
-  "name": "protractor",
+  "name": "psl-workshop-protractor",
   "version": "1.0.0",
   "description": "This is a Workshop about Protractor",
   "main": "index.js",
   "scripts": {
     "clean": "rm -rf dist",
     "prebuild": "npm run clean",
-    "build": "tsc",
+    "build": "node_modules/.bin/tsc",
```

Comment:
why do you do this change

Reply:
the thing is it was showing this error : 
![image](https://user-images.githubusercontent.com/9938092/31288845-37ea6f80-aa8c-11e7-8b54-9928b192e415.png)

so, then i realize that i had not installed the protractor locally, and when i try to do it, it shows me an error (Refusing to install protractor as a dependency of itself), so i just change the name on package.json and it let install the protractor locally, then by itself the system changes the package.json with that value and the new protractor dependency, i test it and it works, i don´t know if i should continue with this build script. PD : i am on windows 10